src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -339,18 +339,24 @@ private void ChooseActions(in Entity entity, ref MicrobeAI ai, ref CompoundAbsor
             ai.ATPThreshold = 0.95f * speciesFocus / Constants.MAX_SPECIES_FOCUS;
         }
 
-        if (ai.ATPThreshold > 0.0f)
+        if (ai.ATPThreshold > Mathf.Epsilon)
```

Comment:
We have our own epsilon value in MathUtils that is used rather than the engine provided value.

Reply:
right, forgot about that!